features/home/src/main/kotlin/com/teacherworkout/features/home/SearchView.kt

```diff
@@ -0,0 +1,65 @@
+package com.teacherworkout.features.home
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.Icon
+import androidx.compose.material.OutlinedTextField
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.material.icons.filled.Search
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.TextFieldValue
+
+@Composable
+fun SearchView(
+    modifier: Modifier = Modifier,
+    value: MutableState<TextFieldValue>,
+    onValueChange: (TextFieldValue) -> Unit,
+    onClear: () -> Unit,
+    placeholderText: String
+) {
+    val corner8dp = dimensionResource(id = R.dimen.space_16dp)
```

Comment:
Pretty confusing to have a variable called 'corner8dp' which actually has the size of 16dp. Maybe we can think of better ways to rename this?

Reply:
Oopsie. It really should have been 8dp.